components/_expanders/accordion-expander-nested.hbs

```diff
@@ -0,0 +1,134 @@
+<div class="row">
+    <div class="msds-expander msds-bg-gray-gradient-2">
+        <div class="col-12 col-md-3">
+            <div class="msds-expander__header">
+                <div class="msds-expander__header-icon msds-contact-illustration" title="Contact Illustration">
+                </div>
+                <div
+                    class="msds-expander__header-headline msds-expander--hide-mobile-element msds-text-header-2 msds-text-gray-10">
+                    Frequently asked questions
+                </div>
+            </div>
+        </div>
+        <div class="col-12 col-md-9">
+            <div
+                class="msds-expander__header-headline msds-expander--hide-desktop-element msds-text-header-2 msds-text-gray-10">
+                Frequently asked questions
+            </div>
+            <ul class="msds-expander__body ">
+                <li class="msds-expander__list-item" tabindex="0">
```

Comment:
I would extract expander-item out into its  own block instead of using as an element. Similar to what they have done here with the Input http://getbem.com/introduction/ this will clean up the code a bit.

Reply:
I disagree,I see the expander-item as a element of the expander